plutus-contract/test/Spec/Contract/Tx/Constraints/RequiredSigner.hs

```diff
@@ -40,6 +51,8 @@ tests =
         , otherWalletNoSigningProcess
         , phase2FailureMustBeSignedBy
         , withoutOffChainMustBeSignedBy
+        , cardanoTxOwnWallet -- When we'll have enough constraints, reuse the previous tests
```

Comment:
move comment to new line above so that it clearly applies to all tx-constraint tests below it